ols/app/endpoints/ols.py

```diff
@@ -145,8 +145,8 @@ def generate_response(
                     llm_response["history_truncated"],
                 )
             except Exception as summarizer_error:
-                logger.error("Error while obtaining answer for user question")
-                logger.error(summarizer_error)
+                logger.exception("Error while obtaining answer for user question")
+                logger.exception(summarizer_error)
```

Comment:
i agree we want the traceback, but do we really want it twice?

Reply:
aaah, good catch!. Yes the 1st might be just `error`

Reply:
updated @bparees